locale/en/default.po

```diff
@@ -152,3 +152,12 @@ msgstr "Reviewers"
 
 msgid "default.groups.abbrev.externalReviewer"
 msgstr "R"
+
+msgid "default.submission.step.reviewerSuggestions"
+msgstr ""
+"<p>When submitting your article, you have the option to suggest 2/3 potential "
```

Comment:
is this "2/3 potential reviewers" the correct wording here?

Reply:
This is taken from what provided in the design but I think you are right , better to say it like `you have the option to suggest several potential reviewers`  .  Or you have anything better in mind .